subsys/zigbee/osif/zb_nrf_serial_logger.c

```diff
@@ -107,7 +107,7 @@ void zb_osif_serial_logger_put_bytes(const zb_uint8_t *buf, zb_short_t len)
 	}
 
 	if (k_sem_take(&ringbuf_sem, K_FOREVER)) {
-		LOG_ERR("Couldn't take semaphore, dropping %u bytes", len);
```

Comment:
Could you elaborate, why this message should be at the "Debug" level?

Reply:
You're right, changing this doesn't make sense. It was other log message that it was to get rid of but logging level for this module has been changed to `info` - no need for change, nor this PR to exist